### DISCUSSION TITLE
Is MS Store version a UWP?

### DISCUSSION THREAD

**Discussion by e-t-l** [infinitepower18/WSA-Sideloader#123] [Support] [answered]
It's installed with an MSIX file into the WindowsApps directory, which means it's a UWP, right? The Microsoft Store says under Permissions that it "uses all system resources," which usually indicates that an app is not a UWP. If it is, could we limit it to only requesting necessary permissions? This is security best practice.

**Comment by infinitepower18** [comment, marked as answer]
No, this app is not UWP. It's a traditional Win32 application packaged into MSIX format. Pretty much all traditional programs on the MS Store will show "uses all system resources" on the store listing.

There is [app isolation](https://github.com/microsoft/win32-app-isolation) but as far as I'm aware, this requires a Windows 11 version higher than 22621 which is the current public release.

Also worth pointing out that Microsoft has deprecated UWP and are promoting WinUI 3 and WPF. WinUI 3 apps are not sandboxed AFAIK.
